website/templates/emails/pending_embargo.txt.mako

```diff
@@ -0,0 +1,20 @@
+Hello ${user.fullname},
```

Comment:
My suggestions:

change the first line to:
This email is to notify you that another administrator on one of your projects has requested an embargoed registration.


and then leave the rest the same.